models/engine/file_storage.py

```diff
@@ -78,7 +78,7 @@ def get(self, cls, id):
         """returns object based on the class name"""
         dict1 = self.all(cls)
         list1 = dict1.values()
-            for item in list1:
+        for item in list1:
```

Comment:
What item and list1 refer to?

Reply:
list1 is a lit with the values of the dictionary. item is each of the values, this variable is used to iterate over the list